Assignment-3/Marksheet.java

```diff
@@ -0,0 +1,136 @@
+import java.util.Scanner;
+class Functions {
+	private double[] marks;
+	private int size;
+	
+	/**
+	 * This constructor for taking number of students
+	 * @param size > 0
+	 */
+	public Functions(int size) {
+		this.size=size;
+	}
+	/**
+	 * This function gives average of marks 
+	 * @param marks should be between 0 and 100 both
+	 * @return average of marks
+	 * @throws ArithmeticException, if user has entered wrong values
+	 */
+	

```

Comment:
remove this space and similarly in all the places. Space should be in-between function and not between their relevant comments.